master/src/main/scala/org/apache/celeborn/service/deploy/master/http/api/v1/ApplicationResource.scala

```diff
@@ -96,41 +108,18 @@ class ApplicationResource extends ApiRequestContext {
     new HostnamesResponse().hostnames(statusSystem.hostnameSet.asScala.toSeq.asJava)
   }
 
-  @Path("/reviseLostShuffles")
+  @Path("/revise_lost_shuffles")
```

Comment:
Is there any standard of path pattern? If yes, could you add some description in `CONTRIBUTING.md` to tell contributor this standard.

Reply:
update the readme

Reply:
<img width="935" alt="image" src="https://github.com/user-attachments/assets/234e0e33-7709-48b4-ba88-93c51c6abe84">